.github/CODEOWNERS

```diff
@@ -34,10 +34,10 @@
 /src/mono/mono/metadata/thread* @lateralusX @lambdageek
 /src/mono/mono/metadata/w32* @lateralusX @lambdageek
 
-/src/mono/mono/mini @vargaz @lambdageek @SamMonoRT @imhameed
+/src/mono/mono/mini @vargaz @lambdageek @SamMonoRT
 /src/mono/mono/mini/*cfgdump* @vargaz
-/src/mono/mono/mini/*exceptions* @vargaz @BrzVlad @imhameed
-/src/mono/mono/mini/*llvm* @vargaz @imhameed @EgorBo
+/src/mono/mono/mini/*exceptions* @vargaz @BrzVlad
+/src/mono/mono/mini/*llvm* @vargaz @EgorBo
 /src/mono/mono/mini/*ppc* @vargaz
 /src/mono/mono/mini/*profiler* @BrzVlad @lambdageek
 /src/mono/mono/mini/*riscv* @alexrp
```

Comment:
Can you confirm that this line is still correct?

Reply:
Specifically, alexrp shows with an error that they don't have permissions.

Reply:
seems right, Egor moved to Juliee's team but has been helping with reviews as necessary. We can leave it that way

Reply:
> Specifically, alexrp shows with an error that they don't have permissions.

updated that, thanks for catching that.